network-primitives/scripts/prune_docker_system.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+systemctl stop subspace.service
```

Comment:
BTW, shouldn't we delete the service afterwards. It will try to start after reboot.